src/components/domain/TimeTable/index.tsx

```diff
@@ -64,6 +64,16 @@ const TimeTable = ({ isToday, timeSlot, onModalOpen, onModalClose }: Props) => {
     [step, handleSetCurrentBlock, handleSetTime, onModalOpen]
   );
 
+  const handleClickReservationMarker = useCallback(
+    (reservationId: number) => {
```

Comment:
후에는 reservationId의 number을 APIReservation의 id를 뽑으면 좋을 거 같아요
```suggestion
    (reservationId: APIReservation["id"]) => {
```
interface의 property type을 구하는 방법이 궁금해서 찾아보니 위와 같이 사용하면 할 수 있다고 하네요~!

![image](https://user-images.githubusercontent.com/61593290/151661784-a4ac456f-3221-4d0b-a248-c66b8291cca0.png)

Reply:
이 부분 타이핑할 때 신경써서 고려해볼게요!! 일단 타이핑은 뒤로 미뤄두겠습니다